### PR TITLE
ML more form styling

### PR DESCRIPTION
**Pull request by m-lamarre**

### CODE REVIEW

File: app/views/devise/passwords/new.html.slim [hunk 1]
```diff
@@ -4,12 +4,14 @@
   .medium-6.medium-centered.large-4.large-centered.columns
     .row.column.log-in-form
       h2.text-center
-        | Forgot your password?
+        | Forgot Your Password?
       = simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
         = f.error_notification
         .form-inputs
           = f.input :email, required: true, autofocus: true
+        br
```
Comment (Keyoung-n) [inline app/views/devise/passwords/new.html.slim]: Why not have .gap classes that way you can change the gaps later. best practice to separate styling and markup

File: config/application.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+gmail_username: 'hypertube.thewildcards@gmail.com'
```
Comment (Keyoung-n) [inline config/application.yml]: This shouldn't be pushed up

Comment (Keyoung-n) [inline config/application.yml]: merge development in to your branch and git rm this file and re add the username and password to fix this

File: app/models/user.rb [hunk 3]
```diff
@@ -2,6 +2,7 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   validates_format_of :username, with: /^[a-zA-Z0-9_\.]*$/, multiline: true
+  validates_presence_of :username
```
Comment (Keyoung-n) [inline app/models/user.rb]: My bad thank you. >.<

File: config/application.yml (hunk 2 above)
Comment (m-lamarre) [inline config/application.yml]: Ok, so I should just delete this file?

And I'm not entirely sure what **username and password** I'm supposed to **re-add**? 🤔 

screen shot 2017-04-28 at 9 58 08 am

File: app/views/devise/passwords/new.html.slim (hunk 1 above)
Comment (m-lamarre) [inline app/views/devise/passwords/new.html.slim]: I understand what you're saying, separating styling and mark-up.
But the `<br>` tag is an HTML element that creates that line break.
Do you think it'll be better to add a _.gap_ class, for something that is handled by the _single line break tag_?

screen shot 2017-04-28 at 9 37 46 am

Comment (Keyoung-n) [inline app/views/devise/passwords/new.html.slim]: yup so that way if you want to change the the spacing later you can and it'll effect the hole site.

Comment (m-lamarre) [inline app/views/devise/passwords/new.html.slim]: Ok, I'll use this afternoon to change all the `br` tags to `line height` measurements instead 👍

**Pull request merged**